src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -55,19 +54,12 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
    * Schedule work according to the schedule of the `backoff`.
    */
   private def scheduleWork(work: => Unit): Unit = {
-    val interval = backoff.nextBackOffMillis().millis
-    context.system.scheduler.scheduleOnce(interval) {
+    val interval = backendCall.pollBackoff.next.backoffMillis
+    context.system.scheduler.scheduleOnce(interval.millis) {
       work
     }
```

Comment:
Bummer.  I use FSM state data to get around this sort of thing in the other actors, but CEA is not FSM (and I'm not suggesting you need to change it now :smile: ).